testing/iframe.js

```diff
@@ -275,11 +275,15 @@ const IFRAME_STUB_URL =
  * See /test/fixtures/served/iframe-stub.html for implementation.
  *
  * @param win {!Window}
+ * @param opt_beforeAttachToDom {function=}
```

Comment:
line 285 calls the function with one param. that means we expect a function with one param.